en/advanced/system_tunes.md

```diff
@@ -0,0 +1,83 @@
+
+PX4 also has a module that can be used to play (test) the default tunes or a user defined tune.
+
+This topic provides general guidance on how to create your own tunes and add to/replace the system notification tones/tunes.
+
+
+## Creating Tunes
+
+Tune strings are defined using [ANSI Music notation](http://artscene.textfiles.com/ansimusic/information/ansimtech.txt).
+
+> **Tip** More information about the format can be found in [QBasic PLAY statement](https://en.wikibooks.org/wiki/QBasic/Appendix#PLAY) (Wikibooks) and has been reproduced in [tune_definition.desc](https://github.com/PX4/Firmware/blob/master/src/lib/tunes/tune_definition.desc).
+
+The easiest way to create a new tune is to use a music editor.
+This allows you to edit the music and play it back on your computer, then export it to a format that can be played by PX4.
+
+ANSI music was popular in the days of ANSI BBS systems, and so the best editing tools are DOS utilities.
+On Windows, one option is to use *Melody Master* within *Dosbox*.
+
+The steps for using the software are:
+
+1. Download [DosBox](http://www.dosbox.com/) and install the app
+1. Download [Melody Master](ftp://archives.thebbs.org/ansi_utilities/melody21.zip) and unzip into a new directory
+1. Open the *Dosbox* console
+1. Mount the melody master directory in Dosbox as shown below:
+   ```
+   mount c C:\<path_to_directory\Melody21
+   ```
+1. Start *Melody Master* with the following commands
+   ```
+   c:
+   start
+   ```
+1. You will then have the option to click through a few screens until *Melody Master* is displayed:
+   ![Melody Master 2.1](../../assets/tunes/tunes_melody_master_2_1.jpg)
+   
+   The lower half of the screen provides helpful advice on keyboard shortcuts for using the tool (arrows for moving in stave, and numbers for selecting the note length, etc.).
+1. When you're ready to save the music:
+   - Press **F2** to give the tune a name and save it in the */Music* sub folder of your Melody Master installation.
+   - Press **F7**, the scroll down the list of output formats on the right to get to ANSI.
+     The file will be exported to the *root* of the Melody Master directory (with the same name and a file-type specific extension).
+1. Open the file.
+   The output might look like this:
+   
+   ![ANSI Output from file](../../assets/tunes/tune_musicmaker_ansi_output.png)
+
+1. The string that can be played in PX4 is the bit between `MNT` and `P64`: `150L1O3DL16CL32<B>C<AEL16A`
+
+
+## Testing Tunes
+
+When you're ready to try it out a new tune on PX4, use the [tune_control](../middleware/modules_system.md#tunecontrol) library.
+For example, to test the tune we "created" above you would enter the following command on a console or shell (e.g. the [MAVLink Shell](../debug/system_console.md#mavlink_shell)):
+```sh
+tune_control play -m "150L1O3DL16CL32<B>C<AEL16A"
+```
+
+> **Note** Out of the box, the tune_control is only present on real hardware (not the simulator).
```

Comment:
@julianoes FYI, this is still true, but I'm pretty sure I saw a PR in progress to enable support on SITL.